application/src/main/java/dev/stratospheric/todoapp/todo/NotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package dev.stratospheric.todoapp.todo;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.NOT_FOUND)
+public class NotFoundException extends RuntimeException{
```

Comment:
I thought it's better to get 404 than 500 if the todo is not found or from someone else